rusty_pizza_server/src/order_model/order.rs

```diff
@@ -653,4 +668,63 @@ mod tests {
         //Then
         assert_eq!(Err(expected_change), calculated_change);
     }
+
+    #[rstest]
+    #[case(Money::new(5, 50), Money::new(7, 42), 0, Some(Money::new(5, 50)))]
+    #[case(Money::new(2, 34), Money::new(6, 78), 1, Some(Money::new(6, 78)))]
+    #[case(Money::new(3, 21), Money::new(9, 99), 2, None)]
```

Comment:
I have to admit that I was skeptical whether this syntax is an improvement, but it definitely is :+1: